docs/how-to/work-with-pinning-services.md

```diff
@@ -32,7 +32,7 @@ To add and use a remote pinning service directly in IPFS, you'll first need to h
 
 ### IPFS Desktop or IPFS Web UI
 
-You can add your favorite pinning service(s) to IPFS Desktop/Web UI directly, enabling you to pin and unpin items from the Files screen in the same way as as you do local pins.
+You can add your favorite pinning service(s) to IPFS Desktop/Web UI directly, enabling you to pin and unpin items from the Files screen in the same way you do with local pins.
```

Comment:
So, the grammar feels off on both the new and old sentence. Suggestion is to remove "in", and "with", since a "local pin" is a thing you do, not do with. So it would read:

"You can add your favorite pinning service(s) to IPFS Desktop/Web UI directly, enabling you to pin and unpin items from the Files screen the same way you do local pins."

Alternatively, just remove "with" and I think it still reads correctly:

"You can add your favorite pinning service(s) to IPFS Desktop/Web UI directly, enabling you to pin and unpin items from the Files screen in the same way you do local pins."

Reply:
I'll replace "with" with "add", that should make more sense.